frontend/packages/dev-console/src/components/topology/podring-utils.ts

```diff
@@ -0,0 +1,58 @@
+import * as _ from 'lodash';
+import { TransformTopologyData } from './topology-utils';
+
+export default class TransformPodRingData {
```

Comment:
Would prefer to see this as a single function or set of utility functions instead of a class.
Should be easier to do once the over fetching of data is cleaned up.